transport/nio/src/main/java/org/kaazing/gateway/transport/nio/internal/NioDatagramAcceptor.java

```diff
@@ -85,6 +89,20 @@ protected IoAcceptorEx initAcceptor(final IoSessionInitializer<? extends IoFutur
             logger.debug("MAXIMUM_READ_BUFFER_SIZE setting for UDP acceptor: {}", maximumReadBufferSize);
         }
 
+        String bossRecvBufferSize = UDP_SERVER_RECEIVE_BUFFER_SIZE.getProperty(configuration);
+        acceptor.setReceiveBufferSize(bossRecvBufferSize);
+        logger.debug("UDP_SERVER_RECEIVE_BUFFER_SIZE setting for UDP acceptor: {}", bossRecvBufferSize);
+
+        String bossSendBufferSize = UDP_SERVER_SEND_BUFFER_SIZE.getProperty(configuration);
+        acceptor.setSendBufferSize(bossSendBufferSize);
+        logger.debug("UDP_SERVER_SEND_BUFFER_SIZE setting for UDP acceptor: {}", bossSendBufferSize);
+
+        int workerRecvBufferSize = UDP_CLIENT_RECEIVE_BUFFER_SIZE.getIntProperty(configuration);
+        acceptor.getSessionConfig().setReceiveBufferSize(workerRecvBufferSize);
+
+        int workerSendBufferSize = UDP_CLIENT_SEND_BUFFER_SIZE.getIntProperty(configuration);
+        acceptor.getSessionConfig().setSendBufferSize(workerSendBufferSize);
```

Comment:
Similar question to the test, why do we need to set the values at both acceptor and sessionConfig level?